jerry-ext/module/jerry-module.c

```diff
@@ -0,0 +1,194 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jmem.h"
+#include "jerryscript.h"
+#include "jerry-module.h"
+
+static void
+jerryx_module_header_insert (jerryx_module_header_t *link,
+                            jerryx_module_header_t **root)
```

Comment:
Based on my understanding, `root` is usually for tree, and `head`/`tail` is for linked list.
What about the signature
```c
static void
jerryx_module_list_insert_head (jerryx_module_header_t *item_p,
                                jerryx_module_header_t **head_pp)
```